smart_contracts/contract/src/contract_api/system.rs

```diff
@@ -180,6 +187,18 @@ pub fn transfer_from_purse_to_account(
     TransferredTo::result_from(transferred_to_value)
 }
 
+/// Transfers `amount` of motes from `source` purse to a `target` public key. If an account
+/// referenced by a `target` public key does not exist it will be created.
```

Comment:
```suggestion
/// Transfers `amount` of motes from `source` to the main purse of `target`.  If the account
/// referenced by `target` does not exist, it will be created.
```